README.md

```diff
@@ -510,15 +511,8 @@ In place of common selectors like class, id or attribute we can use `document.qu
   $newEl.insertAfter(queryString);
 
   // Native
-  function insertAfter(newEl, queryString) {
-    const parent = document.querySelector(queryString).parentNode;
-
-    if (parent.lastChild === newEl) {
-      parent.appendChild(newEl);
-    } else {
-      parent.insertBefore(newEl, parent.nextSibling);
-    }
-  },
+  const target = document.querySelector(queryString);
+  target.parentNode.insertBefore(newEl, target.nextSibling);
```

Comment:
This looks great :clap:.
I just checked to make sure that if `target.nextSibling` return `null` as the last child, this just works as `appendChild`, brilliant.